configure.ac

```diff
@@ -1370,9 +1380,9 @@ echo "  build os      = $BUILD_OS"
 echo
 echo "  CC            = $CC"
 echo "  CFLAGS        = $CFLAGS"
-echo "  CPPFLAGS      = $CPPFLAGS"
+echo "  CPPFLAGS      = $CPPFLAGS $DEBUG_CPPFLAGS $HARDENED_CPPFLAGS"
```

Comment:
CPPFLAGS and CXXFLAGS should probably be printed after the other flags instead of before, since they will override them.